packages/flet/lib/src/controls/cupertino_picker.dart

```diff
@@ -10,43 +10,21 @@ import 'control_widget.dart';
 class CupertinoPickerControl extends StatefulWidget {
   final Control control;
 
-  const CupertinoPickerControl({super.key, required this.control});
+  CupertinoPickerControl({Key? key, required this.control})
+      : super(key: ValueKey("control_${control.id}"));
 
   @override
   State<CupertinoPickerControl> createState() => _CupertinoPickerControlState();
 }
 
 class _CupertinoPickerControlState extends State<CupertinoPickerControl> {
-  int _index = 0;
-  int previousIndex = 0;
-  bool isScrollUp = false;
-  bool isScrollDown = true;
   FixedExtentScrollController scrollController = FixedExtentScrollController();
 
   @override
   void initState() {
     super.initState();
     scrollController = FixedExtentScrollController(
-        initialItem: widget.control.getInt("selectedIndex", _index)!);
-    scrollController.addListener(_manageScroll);
-  }
-
-  void _manageScroll() {
-    // https://stackoverflow.com/a/75283541
-    // Fixes https://github.com/flet-dev/flet/issues/3649
-    if (previousIndex != scrollController.selectedItem) {
-      isScrollDown = previousIndex < scrollController.selectedItem;
-      isScrollUp = previousIndex > scrollController.selectedItem;
-
-      var previousIndexTemp = previousIndex;
-      previousIndex = scrollController.selectedItem;
-
-      if (isScrollUp) {
-        scrollController.jumpToItem(previousIndexTemp - 1);
-      } else if (isScrollDown) {
-        scrollController.jumpToItem(previousIndexTemp + 1);
-      }
-    }
```

Comment:
Can you test that it works on Windows normally?

Reply:
Checked. Looks like it might have been fixed in Flutter:

https://github.com/user-attachments/assets/2d92575b-9404-4a0e-8ca6-54f8fd881942